### ISSUE TITLE
training custom data result

### ISSUE THREAD

**Issue by hehewang625327** [ifzhang/FairMOT#414]
Using our own data set by darklabel , the results are shown in the figure
![01697](https://user-images.githubusercontent.com/36059456/125237630-9432a780-e318-11eb-911d-c160f882b83d.jpg)

**Comment by zengjie617789** [comment]
what about the training results. Please post more details, no one can help you otherwise.

**Comment by hehewang625327** [comment]
通过darklabel软件制作了相关数据集，然后将数据格式转成MOT16数据集格式进行训练，结果进行测试的时候发现如图结果。通过训练MOT16数据集进行结果对比，MOT16的训练结果正常。遂进行排查错误，结果发现制作数据集时，darklabel生成的标注文件在转gt.txt文本环节与MOT16数据集的gt.txt的格式不一致。将其修改后进行训练发现结果依旧如图所示。现在猜测是否是视频标注有问题？

**Comment by zengjie617789** [comment]
你训练模型骨干网络是？ “MOT16训练结果正常”是指什么？你的训练数据是自建数据和MOT16？

**Comment by danishmansari** [comment]
Looking at the bounding boxes it seems, you're not taking care of the offset, prediction can center(x, y), width, height.
So get top-left coordinate you need subtract w/2 from center x and h/2 from center y.

**Comment by ret-1** [comment]
> Looking at the bounding boxes it seems, you're not taking care of the offset, prediction can be center(x, y), width and height.
> So to get top-left coordinate you need subtract w/2 from center x and h/2 from center y.

Exactly. I had the same issue and finally realized there is something wrong with my annotation.
Thanks!
